### PR TITLE
feat(gatsby-remark-katex): support katex options

### PR DESCRIPTION
**Pull request by sidharthachatterjee**
Add support for passing in KaTeX options (https://github.com/KaTeX/KaTeX/blob/master/docs/options.md)

Fixes https://github.com/gatsbyjs/gatsby/issues/11320

### CODE REVIEW

File: packages/gatsby-remark-katex/README.md [hunk 1]
```diff
@@ -16,7 +16,11 @@ plugins: [
     resolve: `gatsby-transformer-remark`,
     options: {
       plugins: [
-        `gatsby-remark-katex`,
+        resolve: `gatsby-remark-katex`,
+        options: {
+          // Add any KaTeX options from https://github.com/KaTeX/KaTeX/blob/master/docs/options.md here
+          strict: "ignore"
```
Comment (sidharthachatterjee) [inline packages/gatsby-remark-katex/README.md]:
```suggestion
          strict: `ignore`
```

File: packages/gatsby-remark-katex/README.md [hunk 2]
```diff
@@ -16,7 +16,11 @@ plugins: [
     resolve: `gatsby-transformer-remark`,
     options: {
       plugins: [
-        `gatsby-remark-katex`,
+        resolve: `gatsby-remark-katex`,
+        options: {
+          // Add any KaTeX options from https://github.com/KaTeX/KaTeX/blob/master/docs/options.md here
+          strict: `ignore`
+        }
```
Comment (pieh) [inline packages/gatsby-remark-katex/README.md]: we are missing `{}` here to wrap `resolve` and `options` fields

Comment (sidharthachatterjee) [reply packages/gatsby-remark-katex/README.md]: Ah, sorry! Have become incapable of writing code with Prettier 😛

Comment (pieh) [reply packages/gatsby-remark-katex/README.md]: I know the feeling 👍

**Pull request merged**